### PR TITLE
Added stutter feature

### PR DESCRIPTION
**Pull request by zhengliw**
- Added stutter feature :)
- Replaced the relative imports with more "accurate" ones
- .gitignore: added .vscode

### CODE REVIEW

File: uwuify/cli.py [hunk 1]
```diff
@@ -20,12 +20,12 @@
 
 import click
 
-from uwuify import SMILEY, YU, uwu
+from __init__ import SMILEY, YU, uwu, STUTTER
```
Comment (StarrFox) [inline uwuify/cli.py]: this should still be `uwuify`

everything else looks fine though

**Review by StarrFox — CHANGES_REQUESTED** [review]

File: uwuify/cli.py (hunk 1 above)
Comment (dairiki-ren) [reply uwuify/cli.py]: oh, thanks for pointing that out!

**Review by StarrFox — APPROVED** [review]

**Comment by StarrFox** [comment]
thanks for contributing!

**Comment by StarrFox** [comment]
*I changed the name to `--stutter` to better match the other flags; should be live in the 1.2.0 version

**Pull request merged**